### DISCUSSION TITLE
How to set the legend

### DISCUSSION THREAD

**Discussion by Pierre-Moulin** [Ifsttar/I-Simpa#332] [Q&A]
I have some issues with the color map :s 
-Why the legends are adding themselves
-How to come back to the default one ? 
-I would like to set a gradient 70 (green) -->80 (red) dB with an isobar at 75 ... what should I do ?
Thanks for your help :)

![image](https://user-images.githubusercontent.com/106664450/187210427-f5b8da3e-34bd-4efe-b79b-644b26947911.png)

**Comment by cmarchetto7** [comment]
Choosing the color scale, quitting and re-opening the project seems to work with me (surely a bug).
Hope it helps
